DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: SASxport
 Title: Read and Write 'SAS' 'XPORT' Files
-Version: 1.7.0
-Date: 2020-03-06
+Version: 1.7.1
```

Comment:
Since functionality was added, it should be 1.8.0.

Reply:
Updated

Reply:
It still shows 1.7.1 instead of 1.8.0.  Did you miss a push?